File: generator/output/ts-model/category.model.ts
class CategoryModel {
  public id: number = 0;

  // deleted 是否删除
  public deleted?: number = 0;

  // name 分类名称
  public name: string;

  // path 英文名称
  public path: string;

  public created_at: string;

  public updated_at: string;
}

const CategoryModelFields = {
  id: "id",
  deleted: "deleted",
  name: "name",
  path: "path",
  created_at: "created_at",
  updated_at: "updated_at"
};

export {CategoryModel, CategoryModelFields};

File: backend/src/main/java/com/licenseair/backend/domainModel/CategoryModel.java
package com.licenseair.backend.domainModel;

import java.util.List;
import lombok.Data;

import java.lang.String;
import java.lang.Integer;
import java.sql.Timestamp;

/**
* Created by licenseair.com
* Category
*/
@Data()
public class CategoryModel {
  public Long id = null;

  /**
  * 是否删除
  */
  public Integer deleted = null;

  /**
  * 分类名称
  */
  public String name = null;

  /**
  * 英文名称
  */
  public String path = null;

  public Timestamp created_at = null;

  public Timestamp updated_at = null;

}
